Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -4360,6 +4362,87 @@ def typo_metrics_enabled(ttFont):
     yield PASS, "Vertical metrics have not regressed."
 
 
+@check(
+  id = 'com.google.fonts/check/cjk_vertical_metrics',
+  conditions = ['is_cjk_font', 'not remote_styles'],
+  rationale="""CJK fonts have different vertical metrics when compared to
+  Latin fonts. We follow the schema developed by dr Ken Lunde for
+  Source Han Sans and the Noto CJK fonts. Our documentation includes
+  further information: https://github.com/googlefonts/gf-docs/tree/master/Spec#cjk-vertical-metrics
```

Comment:
Please use 4-space indentation on the rationale message to void clipping.
Also do not manually break lines. The reporters will do it for us in a better manner. Just ignore the 80-column limit when writing rationale strings and fontbakery will take care of making it look good in each reporter.

Reply:
Ok thanks!